src/components/tipRecords/Author.vue

```diff
@@ -48,8 +48,7 @@ export default {
     },
   },
   async mounted() {
-    const profile = await Backend.getProfile(this.address);
-    this.name = profile ? profile.preferredChainName : null;
+    this.name = (await Backend.getProfile(this.address)).preferredChainName;
```

Comment:
we can get the names from the names list in the store, the stored name for an account is also the preferredChainName, as the backend sorts them that way when they are globally fetched. This reduces the number of requests.

Reply:
I have created #1134 to address this